src/algebra/lie/basic.lean

```diff
@@ -718,6 +718,13 @@ instance has_mem : has_mem M (lie_submodule R L M) := ⟨λ x N, x ∈ (N : set
 @[simp] lemma mem_carrier (N : lie_submodule R L M) {x : M} : x ∈ N.carrier ↔ x ∈ (N : set M) :=
 iff.rfl
 
+@[simp] lemma mk_coe (S : set M) (h₁ h₂ h₃ h₄) :
+  ((⟨S, h₁, h₂, h₃, h₄⟩ : lie_submodule R L M) : set M) = S := rfl
```

Comment:
I'm surprised we don't have this lemma for any of the other sub_* structures.

Reply:
Me too.

I believe it is just one of those things we haven't really needed yet but I'd love to be corrected if there is a good reason not to have these lemmas.

Reply:
I think it is exactly that. Also the total number of lemmas you would have to add is quadratic in the number of "extends" (`coe_to_set_mk`, `coe_to_submodule_mk`, `coe_to_subalgebra_mk`, ...), so we should probably automate it at some point.